### PR TITLE
development settings

### PR DESCRIPTION
**Pull request by rikeda71**
set development environment

### CODE REVIEW

**Comment by maru0** [comment]
```swagger: "2.0"
info:
  description: "ケーキ検索に関するAPI"
  version: "1.0.0"
  title: "KagawaGourmetMap"
paths:
  /shop:
    get:
      tags:
      - "shop"
      summary: "extract shops info"
      description: "return all shop"
      produces:
      - "application/json"
      parameters:
      - name: "status"
        in: "query"
        description: "shop object that needs to be added to the store"
        required: false
        type: "string"
      responses:
        200:
          description: "successful extract all shop"
          schema:
            type: "array"
            items:
              $ref: "#/definitions/Shop"
  
  /shop/{shopId}:
    get:
      tags:
      - "shop"
      summary: "extract a shop by shopid"
      description: "return shop limited by shopid"
      produces:
      - "application/json"
      parameters:
      - name: shopId
        in: path
        description: "shop object that needs to be added to the store"
        required: true
        type: "string"
      responses:
        200:
          description: "successful extract all shop"
          schema:
            $ref: "#/definitions/Shop"
    patch:
      tags:
      - "shop"
      summary: "extract a shop by shopid"
      description: "return shop"
      produces:
      - "application/json"
      parameters:
      - name: shopId
        in: path
        description: "shop object that needs to be added to the store"
        required: true
        type: "string"
      - name: "shop"
        in: "body"
        description: ""
        required: false
        schema:
          $ref: "#/definitions/Shop"
      responses:
        200:
          description: "successful extract all shop"

  /shop/{shopId}/review:
    post:
      tags:
      - "shop"
      summary: "Add a new shop info"
      description: ""
      produces:
      - "application/json"
      parameters:
      - name: shopId
        in: path
        description: "shop object that needs to be added to the store"
        required: true
        type: "string"
      - name: "review"
        in: "body"
        description: ""
        required: true
        schema:
          $ref: "#/definitions/Reviews"
      responses:
        200:
          description: "successful input"
        405:
          description: "Invalid input"
            

definitions:
  Shop:
    type: "object"
    properties:
      id:
        type: "integer"
        example: "1"
      name:
        type: "string"
        example: "ルーヴ"
      times:
        type: "array"
        items:
          $ref: "#/definitions/Date"
      point:
        type: "number"
        example: "1.0"
      geocode:
        $ref: "#/definitions/LatLng"
  Date:
    type: "object"
    properties:
      day:
        type: "string"
        example: "月曜日"
      startTime:
        type: "string"
        format: date-time
        example: "2017-07-21T13:20:28Z"
      lastTime:
        type: "string"
        format: date-time
        example: "2017-07-21T17:30:28Z"
  Reviews:
    type: "object"
    properties:
      impress:
        type: "string"
        example: "おいしかった"
      eval:
        type: "integer"
        example: "5"
  LatLng:
    properties:
      lat:
        format: float
        type: number
      lng:
        format: float
        type: number
    type: object
```

File: swagger/swagger.yml [hunk 1]
```diff
@@ -1,24 +1,295 @@
...
+        description: "shop object that needs to be added to the store"
+        required: false
+        type: "string"
       responses:
-        '200':    # status code
-          description: A JSON array of user names
-          content:
-            application/json:
+        200:
+          description: "successful extract all shop"
```
Comment (rikeda71) [inline swagger/swagger.yml]:
```suggestion
          description: "successful extract shops"
```

File: swagger/swagger.yml [hunk 2]
```diff
@@ -1,24 +1,295 @@
...
+      tags:
+      - "shop"
+      summary: "extract shops info"
+      description: "return all shop"
+      produces:
+      - "application/json"
+      parameters:
+      - name: "status"
+        in: "query"
+        description: "shop object that needs to be added to the store"
```
Comment (rikeda71) [inline swagger/swagger.yml]:
```suggestion
        description: "get shops"
```

File: swagger/swagger.yml [hunk 3]
```diff
@@ -1,24 +1,295 @@
...
+      tags:
+      - "shop"
+      summary: "extract a shop by shopid"
+      description: "return shop limited by shopid"
+      produces:
+      - "application/json"
+      parameters:
+      - name: shopId
+        in: path
+        description: "shop object that needs to be added to the store"
```
Comment (rikeda71) [inline swagger/swagger.yml]:
```suggestion
        description: "the shop id associated with the shop"
```

File: swagger/swagger.yml [hunk 4]
```diff
@@ -1,24 +1,295 @@
...
+        required: true
+        type: "string"
+      responses:
+        200:
+          description: "successful extract all shop"
```
Comment (rikeda71) [inline swagger/swagger.yml]:
```suggestion
          description: "successful extract a shop"
```

File: swagger/swagger.yml [hunk 5]
```diff
@@ -1,24 +1,295 @@
...
+      tags:
+      - "shop"
+      summary: "extract a shop by shopid"
+      description: "return shop"
+      produces:
+      - "application/json"
+      parameters:
+      - name: shopId
+        in: path
+        description: "shop object that needs to be added to the store"
```
Comment (rikeda71) [inline swagger/swagger.yml]:
```suggestion
        description: "the shop id associated with the shop"
```

File: swagger/swagger.yml [hunk 6]
```diff
@@ -1,24 +1,295 @@
...
+        required: true
+        type: "string"
+      - name: "shop"
+        in: "body"
+        description: ""
+        required: false
```
Comment (rikeda71) [inline swagger/swagger.yml]:
```suggestion
        required: true
```

File: swagger/swagger.yml [hunk 7]
```diff
@@ -1,24 +1,295 @@
...
+        schema:
+          $ref: "#/definitions/Shop"
+      responses:
+        200:
+          description: "successful extract all shop"
```
Comment (rikeda71) [inline swagger/swagger.yml]:
```suggestion
          description: "successful patch the shop"
```

File: swagger/swagger.yml [hunk 8]
```diff
@@ -1,24 +1,295 @@
...
+            type: "array"
+            items:
+              $ref: "#/definitions/Shop"
+  
+  /shop/{shopId}:
+    get:
+      tags:
+      - "shop"
+      summary: "extract a shop by shopid"
+      description: "return shop limited by shopid"
```
Comment (rikeda71) [inline swagger/swagger.yml]:
```suggestion
      description: "return shop by shopid"
```

File: swagger/swagger.yml [hunk 9]
```diff
@@ -1,24 +1,295 @@
...
+          $ref: "#/definitions/Shop"
+      responses:
+        200:
+          description: "successful extract all shop"
+
+  /shop/{shopId}/review:
+    post:
+      tags:
+      - "shop"
+      summary: "Add a new shop info"
```
Comment (rikeda71) [inline swagger/swagger.yml]:
```suggestion
      summary: "Add a new review for the shop"
```

File: swagger/swagger.yml [hunk 10]
```diff
@@ -1,24 +1,295 @@
...
+      description: ""
+      produces:
+      - "application/json"
+      parameters:
+      - name: shopId
+        in: path
+        description: "shop object that needs to be added to the store"
```
Comment (rikeda71) [inline swagger/swagger.yml]:
```suggestion
        description: "the shop id associated with the shop"
```

File: swagger/swagger.yml [hunk 11]
```diff
@@ -1,24 +1,295 @@
...
+        get:
+          tags:
+          - "shop"
+          summary: "extract shops info"
+          description: "return all shop"
+          produces:
+          - "application/json"
+          parameters:
+          - name: "status"
+            in: "query"
```
Comment (rikeda71) [inline swagger/swagger.yml]: ???

**Review by rikeda71 — COMMENTED** [review]
修正よろ

**Pull request merged**